pyvista/core/utilities/transform.py

```diff
@@ -185,6 +187,24 @@ def __init__(self, trans: TransformLike | None = None):
         if trans is not None:
             self.matrix = trans  # type: ignore[assignment]
 
+    def __repr__(self):
+        """Representation of the transform."""
+
+        def _matrix_repr():
+            repr_ = np.array_repr(self.matrix, precision=4, suppress_small=True)
```

Comment:
Maybe it's best not to truncate the precision and/or suppress small values?
```suggestion
            repr_ = np.array_repr(self.matrix)
```

Reply:
+1 not to